tox.ini

```diff
@@ -0,0 +1,32 @@
+# tox (https://tox.readthedocs.io/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+envlist = py
+
+[testenv]
+# Pure software tests
+# Invoke as `tox`, `tox -e py27` etc.
+deps =
+    -r{toxinidir}/test_requirements.txt
```

Comment:
can the test deps be inlined in the tox file itself?

Reply:
Yes, will do